objectherkenning_openbare_ruimte/on_edge/data_delivery_pipeline/run_data_delivery_pipeline.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import time
+import traceback
+from datetime import datetime
+
+from objectherkenning_openbare_ruimte.on_edge.data_delivery_pipeline.components.data_delivery import (
+    DataDelivery,
+)
+from objectherkenning_openbare_ruimte.settings.luna_logging import setup_luna_logging
+from objectherkenning_openbare_ruimte.settings.settings import (
+    ObjectherkenningOpenbareRuimteSettings,
+)
+
+if __name__ == "__main__":
+    settings = ObjectherkenningOpenbareRuimteSettings.set_from_yaml("config.yml")
+    logging_file_path = f"{settings['logging']['luna_logs_dir']}/data_delivery_pipeline/{datetime.now()}.txt"
```

Comment:
Can we change the log file name to not include colons `:`? This prevents copying the file to non-linux filesystems.